ui/ui-frontend/projects/vitamui-library/src/app/modules/models/units/object-group.interface.ts

```diff
@@ -85,6 +85,7 @@ export interface VersionWithQualifierDto extends VersionDto {
   qualifier: ObjectQualifierType;
   version: number;
   opened: boolean;
+  downloadAllowed: boolean;
```

Comment:
Peut être undefined dans le reste de l'application
```suggestion
  downloadAllowed?: boolean;
```

Reply:
ajouté